backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -635,11 +619,13 @@ public PartialOktaUser findUser(String username) {
         getUserOrThrowError(
             username, "Cannot retrieve Okta user's status with unrecognized username");
 
-    Optional<OrganizationRoleClaims> orgClaims = getOrganizationRoleClaimsForUser(user);
+    List<Group> userGroups = userApi.listUserGroups(user.getId()).stream().toList();
+
+    Optional<OrganizationRoleClaims> orgClaims = convertToOrganizationRoleClaims(userGroups);
```

Comment:
Can these two be replaced by the method `getOrganizationRoleClaimsForUser(User user)`?

Reply:
`userGroups` needs to be accessed again in line 628 `isSiteAdmin(userGroups)` that is why I need to keep reference of that information.